docs/porting/libraries.md

```diff
@@ -1,5 +1,153 @@
+
+You can see what it looks like by opening it up:
+
+```javascript
+{
+    "version":"1.0.0-*",
+    
+    "dependencies":{
+        "NETStandard.Library":"1.0.0-rc2-23728"
+    },
+    
+    "frameworks":{
+        "netstandard1.4"
+    }
+}
+```
+
+The most important thing to note is the version number for `NETStandard`.  This actually corresponds to a version number for the [.NET Platform Standard](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md), which is a standard by which you can make a clean tradeoff between API availability and target availability.  For example, if you wanted your .NET Core library to support Windows Phone targets, you would have to use `netstandard1.2`, which would cut off access to APIs available only in higher versions.
+
+You should evaluate the API availability vs Target availability tradeoff you'd like to make and pick the version of `netstandard` that fits your needs the best.  You may find that the default version picked by the template is best for your needs!
+
+Here's the essentials you need to know about `project.json`:
+
+1. You need to restore packages to generate a `project.lock.json` the first time you create the project.
+
+   In Visual Studio, you can right-click the project and select "Restore Packages".
+   
+   If you're using the .NET CLI, you'll need to navigate to the project folder and type:
+   
+   `$ dotnet restore`
+
+2. Any further changes to the `project.json` will require restoring packages to generate a new `project.lock.json` file.
+3. The `project.lock.json` file is what the compiler reads when you build your project.
+
+To learn more about `project.json`, please refer to [the documentation] on it.
+
+Once you can build your .NET Core project(s) successfully, you can begin porting code over!
+
+## Actually porting your code
+
+Finally, porting the code itself!  Depending on how your .NET Framework code is structured, this could actually be the easiest part of porting over to .NET Core.  How you actually go about doing this will depend highly on the way your code is structured.
+
+### Getting a testing framework for testing as you port
+
+The best way to make sure everything works when you've ported your code is to *test your code as you port it to .NET Core*.  To do this, you'll need to use a testing framework that will build and run tests for .NET Core.  Currently, you have two options:
+
+* Xunit
+   - TODO: Instructions for Visual Studio
+   - TODO: Instructions for .NET CLI
+* Nunit
+  - TODO: Instructions for Visual Studio
+  - TODO: Instructions for .NET CLI
+
+If you've been using Xunit or Nunit to write your tests for your library, great!  This shouldn't be much work.  However, if you've been using the built-in testing framework that ships with Visual Studio, you'll need to migrate your existing tests or write new tests in either Xunit or Nunit.
```

Comment:
You could mention xUnit's porting tool, which will automatically change attributes, etc.